agent/host.go

```diff
@@ -54,7 +55,11 @@ type CPUInfo struct {
 // createRootNode creates a graph.Node based on the host properties and aims to have an unique ID
 func createRootNode(g *graph.Graph) (*graph.Node, error) {
 	hostID := config.GetString("host_id")
-	m := graph.Metadata{"Name": hostID, "Type": "host"}
+	hostname, err := os.Hostname()
+	if err != nil {
+		return nil, err
```

Comment:
I rather you set `hostname = ""` and continue adding the root node, would make it more robust

Reply:
We already `panic` - on purpose - if we can't retrieve the hostname in config/config.go
If we can't get the hostname, I'm pretty sure nothing will work :-)

Reply:
fine